### ISSUE TITLE
ICLR-2019-RANDOM MASK: Towards Robust Convolutional Neural Networks 

### ISSUE THREAD

**Issue by BrambleXu** [BrambleXu/knowledge-graph-learning#105]
**一句话总结：**

对CNN进行random mask，完全取代adversarial training，令CNN获得更好的健壮性。

**资源：**

- [pdf](https://openreview.net/forum?id=SkgkJn05YX)
- [code](

**关键字：**

- dataset: 
- CNN

**笔记：**

In this paper, different from most of the existing methods, we take another approach to tackle the adversarial example problem. In particular, we aim to design a new CNN architecture which by itself enjoys robustness, without appealing to techniques such as adversarial training. To this end, we introduce Random Mask as a new ingredient of CNN. To be specific, we randomly select a set of neurons and remove them from the network before training. Then the architecture of the network is fixed during the training and testing process. Note that Random Mask is different from dropout which randomly masks out neurons in each step during training. In addition, Random Mask can be applied very easily to common CNN structures such as ResNet with only a few changes of code. We find that applying Random Mask to the shallow layers of the network is crucial for robustness. CNNs with properly designed Random Mask are far more robust than standard CNNs. In fact, our experimental results demonstrate that CNNs with Random Mask achieve state-of-the-art results against black-box attacks even when comparing with defense methods using adversarial training. Furthermore, CNNs with Random Mask maintain a high accuracy on normal test data, while low test accuracy is often regarded as a major weakness in many methods designed for achieving robustness.

**模型图：**

![image](https://user-images.githubusercontent.com/10768193/56015811-468ff480-5d35-11e9-9f64-9af827754a07.png)

**结果**：

![image](https://user-images.githubusercontent.com/10768193/56015830-51e32000-5d35-11e9-9fa8-be24b99a3460.png)
